src/app/pages/terms-of-service/terms-of-service.component.html

```diff
@@ -74,8 +81,14 @@ <h3>SECTION 4 - THIRD-PARTY LINKS AND SERVICES</h3>
     evaluating the content or accuracy and we do not warrant and will not have any liability or responsibility for any
     third-party materials or websites, or for any other views, endorsements, materials, products, or services of
     third-parties.</p>
+  <p>While we support the embedding of images via third-party links within messages, we currently cannot guarantee that
+    such images will display as intended, nor that the image will not be removed or otherwise made unavailable by the
+    hosting provider. Responsibility for selecting a third-party service and ensuring that an image displays as intended
```

Comment:
Also add a point about the responsibility of the image remaining available falling on the user

Reply:
...? That's already in here? 

> Responsibility for selecting a third-party service and ensuring that an image displays as intended falls on the user;